addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2246,6 +2246,7 @@ var SnippetsMenu = Widget.extend({
         this.el.classList.toggle('d-none', foldState);
         this.el.ownerDocument.body.classList.toggle('editor_has_snippets', !foldState);
         this.folded = !!foldState;
+        this.options?.wysiwyg?.onSnippetsFoldChange?.(this.folded);
```

Comment:
That is a lot of question marks 😄 

Aren't we sure that we have options? And aren't we sure that if we have a wysiwyg, it *will* contain a "onSnippetsFoldChange"?

Reply:
`snippetsEditor` is instanciated in `wysiwyg.js`, and `onSnippetsFoldChange` is defined in `wysiwyg_iframe.js` patch, meaning that depending on loaded assets, the wysiwyg is not guaranteed to have `onSnippetsFoldChange` function.

other `?.` are indeed useless, I'll remove them :)